vm.go

```diff
@@ -351,6 +353,51 @@ func (vm *VirtualMachineSpec) ExecuteCommand(command string) error {
 	return nil
 }
 
+// RunCommand executes a command on the Virtual Machine
+func (vm *VirtualMachineSpec) RunCommand(command string) (output string, err error) {
+	ctx, cancel := context.WithTimeout(context.Background(), 60*time.Minute)
+	defer cancel()
+	system, err := hcs.OpenComputeSystem(ctx, vm.ID)
+	if err != nil {
+		return "", err
+	}
+	defer system.Close()
+
+	commandArray := strings.Split(command, " ")
+	params := &hcsschema.ProcessParameters{
+		CommandArgs:      commandArray,
+		WorkingDirectory: "/",
+		Environment:      map[string]string{"PATH": "/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin"},
```

Comment:
>/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin [](http://example.com/codeflow?start=47&length=60)

Not sure this is a right approach, VirtualMachine doesn't bind with os type, but this is linux specific.  Should this be caller's responsibility to decide what the env is?

Reply:
approach is fine. need to add logic for OS... will do.